src/components/Subthoughts.js

```diff
@@ -242,6 +244,24 @@ export const Subthoughts = connect(({ cursorBeforeEdit, cursor, contextViews, th
           : showContexts ? getContextsSortedAndRanked(/* subthought() || */headValue(thoughtsRanked))
             : getThoughtsRanked(contextBinding || thoughtsRanked)
 
+      // Ensure that editable newThought is visible.
+      const editIndex = (cursor && children && show) ? children.findIndex(child => {
+        const childPath = getChildPath(child, thoughtsRanked, showContexts)
+        const isEditingPath = subsetThoughts(cursor, childPath)
+        return equalPath(cursor, childPath) || isEditingPath
```

Comment:
Since `subsetThoughts` is inclusive, I don't believe you have to check `equalPath`:

```js
const editIndex = (cursor && children && show) ? children.findIndex(child => {
  const childPath = getChildPath(child, thoughtsRanked, showContexts)
  const isEditingPath = subsetThoughts(cursor, childPath)
  return isEditingPath
})
```

Reply:
Changed this entire logic